src/navigation/_AppSettingsProvider.jsx

```diff
@@ -13,19 +13,43 @@ const propTypes = {
   settingsConfig: settingsConfigPropType.isRequired,
 };
 
+/* Use default locales or fall back on default. terra-toolkit v5.21.0 provides configurations from terraI18n,
+  * so this reduces the number of places users need to define their locales.
+  */
+const locales = typeof TERRA_AGGREGATED_LOCALES === 'object' ? TERRA_AGGREGATED_LOCALES : ['en'];
+const themesConfig = typeof TERRA_THEME_CONFIG === 'object' ? TERRA_THEME_CONFIG : {};
+const scopedThemes = themesConfig?.scoped || [];
+const defaultThemeName = themesConfig?.theme || 'terra-default-theme';
+
+const themesMap = {
```

Comment:
Map theme name to css class... generally one to one except for the default theme